src/index.ts

```diff
@@ -475,10 +480,26 @@ export const types10: RegistryTypes = {
 
 export const types12: RegistryTypes = {
```

Comment:
We might want to bump the node's version?

Reply:
Yeah I was not sure about that. We probably don't want to bump the version every time we merge with `develop`, whereas it makes more sense to do it with `master`, hence with a new runtime upgrade.

Reply:
I guess the name would be the lower end of the version range with which this should work. We have the range definitions below to define exactly what this is valid for

Reply:
So does William's comment refer to this version here, or the node runtime version?